recipe/build.sh

```diff
@@ -9,11 +9,30 @@ if [[ "${target_platform}" == osx-* ]]; then
   export CXXFLAGS="${CXXFLAGS} -DTARGET_OS_OSX=1"
 fi
 
+# For builds that are cross-compiled (aarch64), we need to build a bootstrap system first.
+# https://www.erlang.org/doc/installation_guide/install-cross#Build-and-Install-Procedure_Building-With-configuremake-Directly_Building-a-Bootstrap-System
+if [[ "${CONDA_BUILD_CROSS_COMPILATION}" -eq 1 ]]; then
+
+  BOOTSTRAP_PREFIX="${PREFIX}/../bootstrap"
+  CFLAGS= LDFLAGS= ./configure \
+      --enable-bootstrap-only \
+      --host="${CONDA_TOOLCHAIN_BUILD}" \
+      --prefix="${BOOTSTRAP_PREFIX}"
+
+  make -j $CPU_COUNT
```

Comment:
This is probably unneeded given we already added `--enable-bootstrap-only` to `./configure`, but maybe this is worth trying

```suggestion
  make bootstrap -j $CPU_COUNT
```

Reply:
Good thought, unfortunately did not help.